controllers/preflightvalidation_controller_test.go

```diff
@@ -0,0 +1,315 @@
+				func(_ interface{}, list *kmmv1beta1.ModuleList, _ ...interface{}) error {
+					list.Items = []kmmv1beta1.Module{mod}
+					return nil
+				},
+			),
+			mockSU.EXPECT().PreflightPresetVerificationStatus(ctx, &pv, &kmmv1beta1.CRStatus{Name: mod.Name}).Return(nil),
+			mockPreflight.EXPECT().PreflightUpgradeCheck(ctx, &mod, "some kernel version").Return(true, "some message"),
+			mockSU.EXPECT().PreflightSetVerificationStatus(ctx, &pv, &pv.Status.CRStatuses[0], kmmv1beta1.VerificationTrue, "some message").Return(nil),
+			clnt.EXPECT().Get(context.Background(), nsn, &kmmv1beta1.PreflightValidation{}).DoAndReturn(
+				func(_ interface{}, _ interface{}, m *kmmv1beta1.PreflightValidation) error {
+					m.Status.CRStatuses = []kmmv1beta1.CRStatus{kmmv1beta1.CRStatus{Name: mod.Name, VerificationStatus: "True"}}
+					return nil
+				},
+			),
+		)
+
+		res, err := pr.Reconcile(ctx, req)
+
+		Expect(err).To(BeNil())
+		Expect(res).To(Equal(reconcile.Result{}))
+	})
+
+	It("good flow, some not verified", func() {
+		mod := kmmv1beta1.Module{
+			ObjectMeta: metav1.ObjectMeta{
+				Name: "moduleName",
+			},
+		}
+		pv := kmmv1beta1.PreflightValidation{
+			ObjectMeta: metav1.ObjectMeta{
+				Name:      nsn.Name,
+				Namespace: nsn.Namespace,
+			},
+			Spec: kmmv1beta1.PreflightValidationSpec{
+				KernelVersion: "some kernel version",
+			},
+			Status: kmmv1beta1.PreflightValidationStatus{
+				CRStatuses: []kmmv1beta1.CRStatus{kmmv1beta1.CRStatus{Name: mod.Name}},
+			},
+		}
+		gomock.InOrder(
+			clnt.EXPECT().Get(context.Background(), nsn, &kmmv1beta1.PreflightValidation{}).DoAndReturn(
+				func(_ interface{}, _ interface{}, m *kmmv1beta1.PreflightValidation) error {
+					m.ObjectMeta = pv.ObjectMeta
+					m.Spec.KernelVersion = pv.Spec.KernelVersion
+					return nil
+				},
+			),
+			clnt.EXPECT().List(ctx, gomock.Any(), gomock.Any()).DoAndReturn(
+				func(_ interface{}, list *kmmv1beta1.ModuleList, _ ...interface{}) error {
+					list.Items = []kmmv1beta1.Module{mod}
+					return nil
+				},
+			),
+			mockSU.EXPECT().PreflightPresetVerificationStatus(ctx, &pv, &kmmv1beta1.CRStatus{Name: mod.Name}).Return(nil),
+			mockPreflight.EXPECT().PreflightUpgradeCheck(ctx, &mod, "some kernel version").Return(true, "some message"),
+			mockSU.EXPECT().PreflightSetVerificationStatus(ctx, &pv, &pv.Status.CRStatuses[0], kmmv1beta1.VerificationTrue, "some message").Return(nil),
+			clnt.EXPECT().Get(context.Background(), nsn, &kmmv1beta1.PreflightValidation{}).DoAndReturn(
+				func(_ interface{}, _ interface{}, m *kmmv1beta1.PreflightValidation) error {
+					m.Status.CRStatuses = []kmmv1beta1.CRStatus{kmmv1beta1.CRStatus{Name: mod.Name, VerificationStatus: "False"}}
+					return nil
+				},
+			),
+		)
+
+		res, err := pr.Reconcile(ctx, req)
+
+		Expect(err).To(BeNil())
+		Expect(res).To(Equal(reconcile.Result{RequeueAfter: time.Second * reconcileRequeueInSeconds}))
+	})
+
+})
+
+var _ = Describe("GetModulesCheck", func() {
```

Comment:
```suggestion
var _ = Describe("getModulesToCheck", func() {
```

I suggest we keep the `Describe` name identical with the function/method name it is testing.

Reply:
fixed